go/common/node/node.go

```diff
@@ -68,6 +69,13 @@ type Node struct { // nolint: maligned
 	// consensus member.
 	Consensus ConsensusInfo `json:"consensus"`
 
+	// Beacon contains information for this node's participation
+	// in the random beacon protocol.
+	//
+	// TODO: This is optional for now, make mandatory once enough
+	// nodes provide this field.
+	Beacon *BeaconInfo `json:"beacon,omitempty"`
```

Comment:
We should include this in 20.10 in some form so that nodes will already be able to include this field (and the inner `point` as `[]byte`) before there is actually support for the new beacon protocol. Is this structure relatively stable? Otherwise we could also use `Beacon cbor.RawMessage` for now.

Reply:
#3256